### ISSUE TITLE
Title bar height option

### ISSUE THREAD

**Issue by farukbigez** [microsoft/vscode#157229]
After the latest update, title bar height increased and no option to control its height.

I always, use my vscode `window.zoomLevel: 2` and **title bar** wasn't changing according to **zoomLevel**.

If, this is a new accessibility feature then, we should have an option to control **title bar** height.

![Screen Shot 2022-08-05 at 10 57 34 AM](https://user-images.githubusercontent.com/60490540/183032326-cb2bb417-6387-4c02-b918-86740c78d3e4.png)

**Comment by sbatten** [comment]
@farukbigez The title bar will not zoom if all "interactive components" are disabled in the title bar. If you are not using the icons in the top right, you can hide them with a right click. This will resize the title bar to default.

**Comment by sbatten** [comment]
cc @isidorn

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by farukbigez** [comment]
@sbatten Thanks for response. Still, I think we should able to customize title font size and title height.

**Comment by VSCodeTriageBot** [comment]
This feature request has not yet received the 20 community [upvotes](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) it takes to make to our backlog. 10 days to go. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!
